exo-go/src/types/docker_config.go

```diff
@@ -4,7 +4,7 @@ package types
 // docker-compose.yml
 type DockerConfig struct {
 	Image         string            `yaml:",omitempty"`
-	Build         string            `yaml:",omitempty"`
+	Build         map[string]string `yaml:",omitempty"`
```

Comment:
Nevermind. In order to run build a Dockerfile that lives in a different directory than its context, we must specify both `context` (where the Dockerfile is run) and the `dockerfile` path (where it lives). This is the proper implementation.